src/components/Input/index.jsx

```diff
@@ -8,12 +8,23 @@ const Input = (props) => {
   return (
     <MuiPickersUtilsProvider utils={DateFnsUtils}>
       <StyledController {...props} />
+      {props.errors?.[props.name] && (
+        <StyledError style={{ color: "red" }}>
```

Comment:
I think this `style={{ color: "red" }}` is unnecessary.

```suggestion
        <StyledError>
```